doc/sphinx/MIGRATING

```diff
@@ -0,0 +1,238 @@
+
+  to gain access to those definitions in your file (you might need a path prefix). Some especially-important
+  replacements are |Cic|, |Coq|, |CoqIDE|, and |Gallina|, which display those names in small-caps. Please use them,
+  so that they're rendered consistently.
+
+  Similarly, there are some LaTeX macros in preamble.rst that can be useful.
+
+  Conventions:
+
+    - Keywords and other literal text is double-backquoted (e.g. ``Module``, ``Section``, ``(``, ``,``).
+
+    - Metavariables are single-backquotes (e.g. `term`, `ident`)
+
+    - Use the cmd directive for Vernacular commands, like:
+
+      .. cmd:: Set Printing All.
+
+      Within this directive, prefix metavariables (ident, term) with @:
+
+      .. cmd:: Add Printing Let @ident.
+
+      There's also the "cmdv" directive for variants of a command.
+
+    - Use the "exn" and "warn" directives for errors and warnings:
+
+      .. exn:: Something's not right.
+      .. warn:: You shouldn't do that.
+
+    - Use the "example" directive for examples
+
+    - Use the "g" role for inline Gallina, like :g:`fun x => x`
+
+    - Use code blocks for blocks of Gallina. You can use a double-colon at the end of a line::
+
+        your code here
+
+      which prints a single colon, or put the double-colon on a newline.
+
+::
+
+        your other code here
+
+# Making changes to the text
+
+  The goal of the migration is simply to change the storage format from LaTeX to ReStructuredText. The goal is not
+  to make any organizational or other substantive changes to the text. If you do notice nits (misspellings, wrong
+  verb tense, and so on), please do change them. For example, the programming language that Coq is written in is these days
+  called "OCaml", and there are mentions of the older name "Objective Caml" in the reference manual that should be changed.
+
+# Build, view the manual
+
+  In the root directory of your local repo, run "make sphinx". You can view the result in a browser by loading the HTML file
+  associated with your chapter, which will be contained in the directory doc/sphinx/_build/html/ beneath the repo root directory.
+  Make any changes you need until there are no build warnings and the output is perfect. :-)
+
+# Creating pull requests
+
+  When your changes are done, commit them, push to your fork:
+
+   git commit -m "useful commit message" file
+   git push origin sphinx-doc
+
+  (or push to another branch, if you've created one). Then go to your GitHub
+  fork and create a pull request against Maxime's sphinx-doc
+  branch. If your commit is recent, you should see a link on your
+  fork's code page to do that. Otherwise, you may need to go to your
+  branch on GitHub to do that.
+
+# Issues/Questions/Suggestions
+
+  As the migration proceeds, if you have technical issues, have a more general question, or want to suggest something, please contact:
+
+    Paul Steckler <steck@stecksoft.com>
+    Maxime Dénès <maxime.denes@inria.fr>
```

Comment:
Do you want to keep this file, including this, and the reference above to your repository?